opensshFormat.md

```diff
@@ -6,4 +6,4 @@
 <https://bugs.eclipse.org/bugs/show_bug.cgi?id=541703>
 
 * implementation of bcrypt_pbkdf:
-<https://github.com/kruton/jbcrypt/blob/master/jbcrypt/src/main/java/org/mindrot/jbcrypt/BCrypt.java>
\ No newline at end of file
+<https://github.com/kruton/jbcrypt/blob/master/src/main/java/org/mindrot/jbcrypt/BCrypt.java>
```

Comment:
I wonder if we should instead just link to the copy that is maintained directly in this repo instead?